src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/InstanceDescriptor.cs

```diff
@@ -33,18 +29,18 @@ public InstanceDescriptor(MemberInfo member, ICollection arguments) : this(membe
         /// </summary>
         public InstanceDescriptor(MemberInfo member, ICollection arguments, bool isComplete)
         {
-            _member = member;
-            _isComplete = isComplete;
+            MemberInfo = member;
+            IsComplete = isComplete;
 
             if (arguments == null)
             {
-                _arguments = new object[0];
+                Arguments = new object[0];
```

Comment:
Should this be ```Array.Empty<object>()``` instead of ```new object[0]```?  I'm wondering if there's some reason to care about the reference identity in this case.